src/server_manager/web_app/build_action.sh

```diff
@@ -37,24 +37,20 @@ node src/server_manager/install_scripts/build_install_script_ts.node.js \
     build/server_manager/web_app/sh/server_manager/web_app/scripts.tgz > $ROOT_DIR/src/server_manager/install_scripts/do_install_script.ts
 popd > /dev/null
 
-# Compile Typescript
-tsc
+readonly STATIC_DIR=$OUT_DIR/static
+mkdir -p $STATIC_DIR
+
+# Build main.js
+# TODO(fortuna): Have different production and development builds.
+webpack --mode=development --config=$ROOT_DIR/src/server_manager/webpack.config.js
```

Comment:
I changed it so now we only set it if `BUILD_ENV` is defined.

To build for development you can do:
```
BUILD_ENV=development yarn do server_manager/electron_app/run
```

It builds for production by default, which outputs minified code. The default choice is to prevent us from releasing dev code. One nice thing is that we still get a sourcemap, so debugging is easy even in production mode.